flag_bool_with_inverse.go

```diff
@@ -0,0 +1,165 @@
+package cli
+
+import (
+	"flag"
+	"fmt"
+)
+
+type BoolWithInverseFlag interface {
```

Comment:
do we really need this interface ?

Reply:
Not likely,  I'll remove it and add the provided BoolFlag as an exported value in the struct.  Then I can also remove the method to created, and have the flag defined just like any other flag.